projects/instant-ui-library/src/lib/button/base-button/base-button.component.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../styles/global.styles";
+@import "../../styles/variables/constants";
+
+$transition-out: ease-in 0.1s;
+$transition-in: ease-out 0.3s;
+$box-shadow-color: none;
+@mixin boxShadow($color) {
+  box-shadow: 0px 2px 8px $color;
+  &:hover,
+  &:focus {
+    box-shadow: 0px 8px 15px $color;
+  }
+}
+@mixin createButton($shadow-color, $text-color) {
+  background-color: $shadow-color;
+  color: $text-color;
+  @include boxShadow($shadow-color);
+  &:disabled {
+    cursor: not-allowed;
+    background-color: $color--disabled3;
```

Comment:
Have a concern with contrast. Should create ticket for this.

Reply:
[Update disabled state for button component](https://trello.com/c/QmcL0NaF/22-update-disabled-state-for-button-component)